runtime/compiler/trj9/control/CompilationThread.cpp

```diff
@@ -7662,8 +7672,9 @@ TR::CompilationInfoPerThreadBase::compile(
             _compInfo.getPersistentInfo()->getElapsedTime(),
             compiler->getCompThreadID(),
             scratchSegmentProvider.allocationLimit() >> 10,
-            _compInfo.computeFreePhysicalMemory(incomplete) >> 10
+            freePhysicalMemoryB >> 20
```

Comment:
If `incomplete` then the `verboseString` has less parameters, yet `writeLineLocked` still tries to print freePhysicalMemoryB